### DISCUSSION TITLE
Push local branch to remote?

### DISCUSSION THREAD

**Discussion by wgregorian** [Vinzent03/obsidian-git#698] [Problems] [answered]
I am trying to avoid overcomplicating the process. We'd like to enable the branch protection rule in our repo. I made a local branch but faced difficulties pushing it to the remote.  Is there a way to do it with the help of the plug-in? I haven't been able to figure it out.

Suggestions?

**Comment by Vinzent03** [comment, marked as answer]
What exactly did you try? What issues did you encounter? You should be able to create the local branch and just call the push command. It should ask you for a remote and remote branch, where you can name the remote branch name.

**Reply by wgregorian** [reply]
OK, silly me, I walked away, returned, and typed `origin/IN-93` for the remote branch name, and it worked. Sorry for the inconvenience.
